### PR TITLE
Development

### PR DESCRIPTION
**Pull request by docedson**
# Updates #

When a push is made to the main branch, the Docker image will be pushed to GitHub Packages.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "lint": "eslint \"src/**/*.{js,jsx}\"",
     "stylelint": "stylelint src",
     "prettier": "prettier --check \"src\"",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "start": "vue-cli-service serve"
```
Comment (aaronshivers) [inline package.json:12]: 🔧 Can we switch this to `vite`?
https://vitejs.dev/guide/static-deploy.html

**Review by aaronshivers — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (docedson) [reply package.json]: It appears that the vue-cli-service serve wasn't needed, and I deleted it.

File: package.json [hunk 2]
```diff
@@ -43,7 +44,8 @@
     "lint-staged": ">=10",
     "prettier": "2.3.2",
     "stylelint": "^13.13.1",
-    "stylelint-config-standard": "^22.0.0"
+    "stylelint-config-standard": "^22.0.0",
+    "vite": "^2.4.2"
```
Comment (aaronshivers) [inline package.json:48]: 🔧 `vite` is already in the `dependencies` category

```suggestion
    "stylelint-config-standard": "^22.0.0"
```

File: package.json [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.2",
     "clsx": "^1.1.1",
     "enzyme": "^3.11.0",
+    "node.js": "^0.0.1-security",
```
Comment (aaronshivers) [inline package.json:23]: ❓ I'm not sure that we want `node.js`.
https://www.npmjs.com/package/node.js
```suggestion
```

**Review by aaronshivers — CHANGES_REQUESTED** [review]

**Comment by docedson** [comment]
Closing this PR, as it's a part of another PR.

**Pull request closed without merging**